.github/workflows/publish.yml

```diff
@@ -0,0 +1,45 @@
+on:
+  push:
+    paths:
+      - 'data/**'
+    branches:
+      - master
+name: QPP EU Data Publish
+jobs:
+  gitIssueCreation:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Codebase
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+
+      - name: Get changed files in the data folder
+        id: changed-files-specific
+        uses: tj-actions/changed-files@v35
+        with:
+          files: |
+            data/**
+
+      - name: Create Issue in EPCS ETL Repo only when any of the above files change.
```

Comment:
Since we repeat the same information, can we use a matrix strategy for the repos? Also, can we include CMSgov/ in the repo? For the epcs-etl repo, I think it's better if we remove the assignees.

Reply:
matrix strategy worked for peter-evans/create-issue-from-file